Python/initconfig.c

```diff
@@ -83,7 +83,7 @@ static const char usage_3[] = "\
              cumulative time (including nested imports) and self time (excluding\n\
              nested imports). Note that its output may be broken in multi-threaded\n\
              application. Typical usage is python3 -X importtime -c 'import asyncio'\n\
-         -X dev: enable CPython’s “development mode”, introducing additional runtime\n\
+         -X dev: enable CPython's \"development mode\", introducing additional runtime\n\
```

Comment:
Even though conversion would be a different issue, I am curious whether there is any good reason to not use """ quotes for the string instead of "s and '\n\\' and '\"'.

Reply:
It is C.